po/cs.po

```diff
@@ -91,47 +91,49 @@ msgid "Failed to start Software"
 msgstr "Nepodařilo se spustit Software"
 
 #: src/appchooserdialog.c:379
-#, fuzzy, c-format
+#, c-format
 msgid "Choose an application to open the file “%s”."
-msgstr "Nenalezena žádná aplikace, která je schopna otevřít „%s”."
+msgstr "Vyberte aplikaci pro otevření souboru „%s”."
 
 #: src/appchooserdialog.c:384
 msgid "Choose an application."
-msgstr ""
+msgstr "Vyberte aplikaci."
 
 #: src/appchooserdialog.c:401
 #, c-format
 msgid ""
 "No apps installed that can open “%s”. You can find more applications in "
 "Software"
```

Comment:
By the way, possibly a small typo: Missing dot at the end of the sentence.

Reply:
the dialog is still under review